src/compiler/sys/modules/os.ts

```diff
@@ -2,7 +2,7 @@ import { OS_PLATFORM } from '../environment';
 
 export const EOL = '\n';
 
-export const platform = () => (OS_PLATFORM === 'windows' ? 'win32' : OS_PLATFORM);
```

Comment:
We're able to do this as a result of [this comment's context](https://github.com/ionic-team/stencil/pull/3067/files#r710419635)